physics/moninedmf.f

```diff
@@ -1037,6 +1041,17 @@ subroutine hedmf_run (ix,im,km,ntrac,ntcw,dv,du,tau,rtg,          &
             rtg(i,k,1) = rtg(i,k,1)+qtend
             dtsfc(i)   = dtsfc(i)+cont*del(i,k)*ttend
             dqsfc(i)   = dqsfc(i)+conq*del(i,k)*qtend
+            if(lssav .and. ldiag3d) then
```

Comment:
I'm a little confused why tendencies specific to moninedmf are needed. Shouldn't the ones calculated in GFS_PBL_generic suffice? If not, would you want to add ".and. .not. pbl_generic_tend"?

Reply:
The pbl_generic_tend should be in there.  I will fix that.